.github/scripts/cmd/sorter/sorter.go

```diff
@@ -0,0 +1,173 @@
+package sorter
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/coinhall/yacar/internal/enums"
+	"github.com/coinhall/yacarsdk/v2"
+)
+
+func Start(filePaths []string) {
+	sortedJSONsMap := sortYacarJSONs(filePaths)
+	writeYacarJSONs(sortedJSONsMap)
```

Comment:
I think it might be easier to do the `write` in `sort` directly, so the flow will be `read -> sort -> write` in the same goroutine.